testapp/views.py

```diff
@@ -189,19 +169,21 @@ def query_api(term, location):
 
     businesses = response.get('businesses')
 
+    
     if not businesses:
         print(u'No businesses for {0} in {1} found.'.format(term, location))
         return
 
-    business_id = businesses[0]['id']
+    response_dic={}
+    for business in businesses:
+    	business_id=business['id']
+	#print(u'{0} businesses found, querying business info or the top result "{1}" ...'.format(len(businesses), business_id))
```

Comment:
If this is a debug statement I think we can remove it, especially since it's commented out.